vault/resource_ldap_secret_backend.go

```diff
@@ -0,0 +1,684 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/util"
+	"github.com/hashicorp/vault/api"
+	"log"
+)
+
+func ldapSecretBackendResource() *schema.Resource {
+	fields := map[string]*schema.Schema{
```

Comment:
I think this is missing https://developer.hashicorp.com/vault/api-docs/secret/ldap#schema?

Reply:
heh, i just added it a few minutes ago.